Library/ViewModels/ProjectNavigatorViewModel.swift

```diff
@@ -122,10 +126,23 @@ ProjectNavigatorViewModelInputs, ProjectNavigatorViewModelOutputs {
       .map { $0 == .started || $0 == .updating }
       .skipRepeats()
 
+    let swipedToProjectAtIndexFromIndex = self.willTransitionToProjectAtIndexProperty.signal.skipNil()
+      .takePairWhen(
+        self.pageTransitionCompletedFromIndexProperty.signal.skipNil()
+          .filter { completed, _ in completed }
```

Comment:
looks like we do `self.pageTransitionCompletedFromIndexProperty.signal.skipNil().filter { completed, _ in completed }` twice so it could maybe be pulled out into a more descriptive signal